action-ssh-to-runner/action.yaml

```diff
@@ -17,27 +17,70 @@ inputs:
 runs:
   using: "composite"
   steps:
```

Comment:
I see you setting a lot of env variables that which value is fixed and not generated.
I would suggest to set them here:
```suggestion
  env:
    TMATE_CONF: /tmp/tmate.conf
    TMATE_SOCK: /tmp/tmate.sock
  steps:
```

Reply:
Thanks for checking, that's annoying, lol